src/Tizen.Multimedia.Metadata/MetadataEditor/MetadataEditor.cs

```diff
@@ -439,6 +454,7 @@ public string UnsyncLyrics
         /// Internal error.<br/>
         /// </exception>
         /// <exception cref="ObjectDisposedException">The <see cref="MetadataEditor"/> has already been disposed of.</exception>
+        [Obsolete("Deprecated in API12; Will be removed in API14")]
```

Comment:
please move that line down, so it not be in the middle of /// comment block

Reply:
OK I'll fix it.